Course/Course/Presentation/Outline/CourseOutlineView.swift

```diff
@@ -16,24 +16,30 @@ public struct CourseOutlineView: View {
     private let title: String
     private let courseID: String
     private let isVideo: Bool
+    private let dateTabIndex: Int
     
     @State private var openCertificateView: Bool = false
     private var idiom: UIUserInterfaceIdiom { UIDevice.current.userInterfaceIdiom }
 
     @State private var showingDownloads: Bool = false
     @State private var showingVideoDownloadQuality: Bool = false
     @State private var showingNoWifiMessage: Bool = false
+    @Binding private var selection: Int
```

Comment:
How about renaming it to some more meaning name like selectedTabIndex or something similar.

Reply:
The `ScrollSlidingTabBar` also employs an identical naming convention and logic for consistency purposes.